### PR TITLE
Add capybara

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
#### Board:
* [Ticket Notion](https://www.notion.so/rootstrap/Add-feature-system-specs-to-the-admin-site-in-the-Rails-API-Base-just-a-few-489ec65c7c084394918edb1bb4a66045?pvs=4)
---
#### Description:
- Add capybara
- Test admin sign-in

### CODE REVIEW

**Comment by sebastiancaraballo** [comment]
The tests are failing because there's a problem with a dependency. I resolved this issue locally running `yarn install`. How can I solve this here? Help appreciated cc @santib

File: spec/features/admin/sign_in_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Admin SignIn' do
+  let(:admin_user) { create(:admin_user) }
+
+  it 'Admin signs in successfully' do
```
Comment (enzofab91) [inline spec/features/admin/sign_in_spec.rb:6]: Just a suggestion, what if you put this two its inside a `scenario`?

**Review by enzofab91 — APPROVED** [review]
Nice 👏

File: spec/features/admin/sign_in_spec.rb (hunk 1 above)
Comment (sebastiancaraballo) [reply spec/features/admin/sign_in_spec.rb:6]: Captura de pantalla 2023-12-04 a la(s) 15 38 50
Rubocop doesn't like using Capybara specific methods

**Review by JulianPasquale — APPROVED** [review]

File: spec/features/admin/sign_in_spec.rb (hunk 1 above)
Comment (sebastiancaraballo) [reply spec/features/admin/sign_in_spec.rb:6]: Actually, yes. I was [taking a look](https://docs.rubocop.org/rubocop-rspec/cops_rspec_capybara.html#rspeccapybarafeaturemethods) at this and there is a way to enable the usage of Capybara-specific words like `feature`, `scenario`, etc; these are disabled to favor the usage of RSpec-specific words instead. The way I found to allow the usage of Capybara-specific words is adding this config:
```
RSpec/Capybara/FeatureMethods:
  EnabledMethods:
    - feature
    - scenario
    - given
    - background
```
Do we want to enable these words? cc @rootstrap/ruby

Comment (santib) [reply spec/features/admin/sign_in_spec.rb:6]: Is there any reasoning about RuboCop defaults?

**Pull request merged**